cross/Dockerfile

```diff
@@ -45,7 +45,7 @@ RUN curl https://bootstrap.pypa.io/get-pip.py -o get-pip.py && \
 # ------------------------------------------------------------------------------------------------
 # Go support
 RUN GO_VERSION=1.12 && \
-    GO_HASH=b3fcf280ff86558e0559e185b601c9eade0fd24c900b4c63cd14d1d38613e499 && \
```

Comment:
is this a new version of go? should the `GO_VERSION` be updated too?

Reply:
It's the hash for `1.12`, according to git the hash was not updated when `GO_VERSION` was changed from `1.11` to `1.12`. I checked the [golang downloads page](https://golang.org/dl/) and confirmed that that hash is for `1.11`.

Reply:
aah so we were still running 1.11 :-(